ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/ansible_eos_device.py

```diff
@@ -27,6 +27,18 @@
     HAS_ANTA = False
     # Next line to make ansible-test sanity happy
     AntaDevice = object
+except TypeError as e:
+    # Known bug with Python 3.9.7 and Pydantic `conint`, impacting ANTA. Issue: https://github.com/arista-netdevops-community/anta/issues/557
+    if "Interval() takes no arguments" in str(e):
+        msg = (
+            "The ANTA testing framework, used in the AVD eos_validate_state role, has encountered a compatibility issue with Python 3.9.x. "
+            "Please try a different Python version. For assistance or to report your Python version, please refer to the AVD/ANTA GitHub repositories:\n"
```

Comment:
let's add a known version where it works (3.9.13 or >=3.10)

Reply:
Please if we are going to provide recommendation - something above 3.10. Although it definitely works with latest 3.9

Reply:
it is not recommendation really more just a statement that if you are stuck on 3.9 for whatever reason on your system, you are not doomed

Reply:
Updated in 44567d6a0b1e08cd6ecd7ce3a10a29af0b70a73d